libsrc/leddevice/schemas/schema-adalight.json

```diff
@@ -5,7 +5,7 @@
 		"output": {
 			"type": "string",
 			"title":"edt_dev_spec_outputPath_title",
-			"default":"/dev/ttyUSB0",
+			"default":"/dev/ttyACM0",
```

Comment:
I changed this due to the point it is ubuntu/debian convention. A special reason why you change it back? Did i missed something?